pubpub/platform
# Run actions when pubs enter and leave stages

Needs some polish and there may be some caching bugs but this seems to work for me! Make sure to test in the croccroc community which has a pub with the core title field or else the action runs will fail validation.

## Issue(s) Resolved
#313 

## Test Plan

1. Run migrations
2. Visit http://localhost:3000/c/croccroc/stages/manage
3. Add an action to a stage
4. Configure the action if it requires config
5. Create a rule for that action
6. [On the stages page](http://localhost:3000/c/croccroc/stages) move a pub into or out of the stage (depending on the rule you created)
7. You should see the action running as part of the core logs. If you set `LOG_LEVEL=debug` in `jobs/.env.local` you will also see logs from the worker for each event

## Screenshots (if applicable)

## Optional

### Notes/Context/Gotchas

### Supporting Docs
